app/lib/attachment_batch.rb

```diff
@@ -73,7 +73,7 @@ def remove_files
               # Ignore failure to delete a directory, with the same ignored errors
               # as Paperclip
             end
-          when :fog
+          when :fog, :azure
             logger.debug { "Deleting #{attachment.path(style)}" }
             attachment.directory.files.new(key: attachment.path(style)).destroy
```

Comment:
Have you confirmed this works with `:azure`? This code path is pretty specific to `:fog` as far as I can tell.

Reply:
Waiting for this to be confirmed

Reply:
Sorry for the day. I'll confirm and fix as necessary tomorrow.

Reply:
Fixed - pushed a new Azure path for that. Thanks for calling this out.

At this point there isn't a specific batch path in paperclip-azure, so I've just gone for `attachment.destroy`.